sdk/src/client/node_manager/syncing.rs

```diff
@@ -110,14 +111,21 @@ impl ClientInner {
         if let Some(nodes) = network_nodes.get(most_nodes.0) {
             if let Some((info, _node_url)) = nodes.first() {
                 let mut network_info = self.network_info.write().await;
-
-                network_info.tangle_time = info.status.relative_accepted_tangle_time;
                 // Unwrap: We should always have parameters for this version. If we don't we can't recover.
-                network_info.protocol_parameters = info
+                let protocol_parameters = info
                     .protocol_parameters_by_version(PROTOCOL_VERSION)
                     .expect("missing v3 protocol parameters")
                     .parameters
                     .clone();
+                if let Some(network_info) = network_info.as_mut() {
+                    network_info.tangle_time = info.status.relative_accepted_tangle_time;
+                    network_info.protocol_parameters = protocol_parameters;
+                } else {
+                    network_info.replace(NetworkInfo {
+                        protocol_parameters,
+                        tangle_time: info.status.relative_accepted_tangle_time,
+                    });
+                }
```

Comment:
Does anything speak against just always replacing it?

Reply:
no, I don't know why I did this lol